pom.xml

```diff
@@ -33,6 +33,13 @@
             <artifactId>error-reporting-java</artifactId>
             <version>1.0.1</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.exasol/maven-project-version-getter -->
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>maven-project-version-getter</artifactId>
+            <version>1.2.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this used in the code? I didn't see changes using it.

Reply:
It's used in the ExtensionIT class